content/blog/tikv-3.0ga-release-notes.md

```diff
@@ -0,0 +1,164 @@
+
+* **Refining request types.** While TiKV previously supported commands like `BatchGet`, 3.0 brings a new `BatchCommands` request type. This allows TiKV to handle batches of requests of differing kinds (such as `Get` and `Put`), leading to less data on the wire and better performance. We also added support for raw reversed scanning, and `Insert` semantics on prewrite.
+
+* **Reducing write amplification.** Inspired by the great ideas from [WiscKey](https://www.usenix.org/system/files/conference/fast16/fast16-papers-lu.pdf), we implemented Titan, a key-value plugin that improves write performance for scenarios with value sizes greater than 1KB, and relieves write amplification in certain degrees.
+
+You can see all the changes in detail in the [changelog](https://github.com/tikv/tikv/blob/release-3.0/CHANGELOG.md).
+
+## Improved Performance
+
+Using [`go-ycsb`](https://github.com/pingcap/go-ycsb) we benchmarked TiKV 3.0.0 against TiKV 2.1.14. We benchmarked a cluster of 3 TiKV nodes, 1 PD node, and 1 node running YCSB. We used DigitalOcean `s-8vcpu-32gb` size machines, and you can reproduce the benchmark for yourself using the fully automated terraform script [here](https://github.com/Hoverbear/tikv-bench).
+
+The settings for YCSB we used were:
+
+* 1KB Value Size
+* 100 Field Length
+* 10 Field Count
+* 3000 Thread Count (per node, so 9000 total)
+
+According to these results below, and the real-world experience of our customers, we're proud to say that TiKV 3.0.0 is the fastest, most usable TiKV yet!
+
+### Reading the benchmarks & a Disclaimer
+
+Since we used 3 YCSB benchers running simultaneously, you can see the results of benchers 1 through 3 below.
+
+For the results, `Takes(s)`, `Count`, and `OPS` are better if they are higher. The final fields detailing `us` (microseconds) units, lower is better.
+
+These benchmarks were run on a public cloud so you can replicate it yourself. The consequence of this is that public cloud machines have dramatic performance fluctuations, and transient failures. You may experience different results if you replicate it yourself, but on average, you should see relatively similar numbers as below.
+
+### Load
+
+2.1.14:
+
+```
+1 - INSERT - Takes(s): 27.6, Count: 332998, OPS: 12045.0, Avg(us): 244522, Min(us): 31613, Max(us): 1686185, 95th(us): 415000, 99th(us): 0
+2 - INSERT - Takes(s): 28.0, Count: 332999, OPS: 11890.6, Avg(us): 242495, Min(us): 40928, Max(us): 2362282, 95th(us): 389000, 99th(us): 0
+3 - INSERT - Takes(s): 30.2, Count: 333000, OPS: 11030.8, Avg(us): 252949, Min(us): 34649, Max(us): 3452374, 95th(us): 452000, 99th(us): 0
+```
+
+3.0.0:
+
+```
+1 - INSERT - Takes(s): 18.5, Count: 332999, OPS: 17966.8, Avg(us): 158305, Min(us): 1949, Max(us): 3812844, 95th(us): 479000, 99th(us): 1014000
+2 - INSERT - Takes(s): 28.0, Count: 333000, OPS: 11885.8, Avg(us): 238947, Min(us): 16070, Max(us): 1212289, 95th(us): 559000, 99th(us): 923000
```

Comment:
What i concern is why the OPS is much lower for group 2 that compare to group 1 and group 3.